apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentTabs/tabs.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+
+import { Button } from '@latitude-data/web-ui'
+import { ROUTES } from '$/services/routes'
+import Link from 'next/link'
+import { useSelectedLayoutSegment } from 'next/navigation'
+
+export function DocumentTabSelector({
+  projectId,
+  commitUuid,
+  documentUuid,
+}: {
+  documentUuid: string
+  projectId: string
+  commitUuid: string
+}) {
+  const selectedSegment = useSelectedLayoutSegment()
+
+  const pathTo = (path: string) =>
+    ROUTES.projects
+      .detail({ id: Number(projectId) })
+      .commits.detail({ uuid: commitUuid })
+      .documents.detail({ uuid: documentUuid }).root + path
```

Comment:
Don't do this. Do different methods. `.detail({ uuid: documentUuuid })`, `.logs(...)`, ...the point is having it typed. 

Also change it in the other place.